spec/game_runner_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "spec_helper"
+require_relative "../game_runner"
+
+RSpec.describe "GameRunner" do
+  describe "#run" do
+    it "displays an empty board" do
+      BOARD = " __ __ __\n|  |  |  |\n|__|__|__|\n|  |  |  |\n|__|__|__|\n|  |  |  |\n|__|__|__|\n"
```

Comment:
What do you think about using a [squiggly heredoc](https://ruby-doc.org/core-2.6.5/doc/syntax/literals_rdoc.html#label-Here+Documents) here? I find that to be easier to read when newlines are involved.

Reply:
What's the intention of this being a constant?

Reply:
Cool, heredocs are a great solution. 

I was at one point thinking I may need it for multiple tests, but perhaps I can just set it as a variable. I like the variable to make it clear that the test is displaying a board in the assertion.

Reply:
updated here: 9dce024796221375fda75851c2ef7af95989f7f2